src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -180,6 +180,55 @@ void SensorFusion::updateWorld(
     }
 }
 
+static double computeDistanceBetweenBallAndRobot(const Robot &robot, const Ball &ball)
+{
+    Vector robot_position = {robot.position().x(), robot.position().y()};
+    Vector ball_position  = {ball.position().x(), ball.position().y()};
+
+    // using the classic distance formula
+    return (ball_position - robot_position).length();
+}
+
+
+bool SensorFusion::shouldUseRobotBallPositionInsteadOfVision(
```

Comment:
I think `shouldTrustRobotStatus` is a better name

Reply:
agreed